typescript/packages/plugins/steer-protocol/src/parameters.ts

```diff
@@ -0,0 +1,87 @@
+import { z } from "zod";
+
+/**
+ * Parameters schema for depositing tokens into a Steer Smart Pool
+ */
+export const depositParametersSchema = z.object({
```

Comment:
Parameters need to be defined like this:
- https://github.com/goat-sdk/goat/blob/main/typescript/packages/plugins/erc20/src/parameters.ts#L4